ldcbench.html-embd-node/src/main/java/org/dice_research/ldcbench/nodes/htmlembd/SimpleHEComponent.java

```diff
@@ -72,9 +73,9 @@ public void initBeforeDataGeneration() throws Exception {
         String pathTemplate;
         if(singleFileNode) {
             // Create path including the single file name
-            StringBuilder builder = new StringBuilder(CONTENT_EXTRACTION_DIR);
+            StringBuilder builder = new StringBuilder("/" + CONTENT_EXTRACTION_DIR);
             builder.append(SINGLE_FILE_NAME);
-            singleFilePath = builder.toString();
+            singleFilePath = "." + builder.toString();
```

Comment:
Why you do not add the point at the beginning, when you create the builder? :confused:

Reply:
because the dot is only needed for the singleFilePath (for disk access) but not in the URI